managers/coverFetchers/publisher_backlist_cover_fetcher.py

```diff
@@ -0,0 +1,80 @@
+import hashlib
+import io
+import os
+
+import PIL
+import pypdf
+import requests
+import requests.exceptions
+
+from logger import create_log
+from managers.coverFetchers.abstractFetcher import AbstractFetcher
+
+logger = create_log(__name__)
+
+
+class PublisherBacklistCoverFetcher(AbstractFetcher):
+
+    def __init__(self, *args):
+        super().__init__(*args)
+
+        self.session = requests.Session()
+        self.bucket = os.environ["FILE_BUCKET"]
+        self.pdf_url: str | None = None
+
+
+    def hasCover(self):
+        for value, source in self.identifiers:
+            if source != "hathi":
+                continue
+
+            url = (
+                f"https://{self.bucket}.s3.us-east-1.amazonaws.com"
```

Comment:
We are moving towards using the get_file_location method in the digital_assets module. This way it'll work locally as well.